tools/ci/choose_workflows.py

```diff
@@ -41,6 +46,7 @@ def should_run_sr_unit_tests(ctx: CIContext) -> bool:
         build_env=context.build_env.DD_OVERRIDE_RUN_SR_UNIT_TESTS,
         pr_keyword='[x] Run unit tests for Session Replay',
         pr_path_prefixes=[
+            'Datadog/Datadog.xcodeproj/',
```

Comment:
/suggestion we could also add `DatadogInternal` since SR depends on it, wdyt?

Reply:
+1

Reply:
Definitely!